automation/capitanodoc/capitanodoc.ts

```diff
@@ -82,6 +82,14 @@ const capitanoDoc = {
 				'build/commands/device/shutdown.js',
 			],
 		},
+		{
+			title: 'Releases',
+			files: [
+				'build/commands/releases.js',
+				'build/commands/release/index.js',
+				'build/commands/release/finalize.js',
```

Comment:
The `doc/cli.markdown` file was missing reference documentation for the `releases`, `release` and `release finalize` commands. Adding these lines fixes it.